python/cudf/cudf/core/dataframe.py

```diff
@@ -7425,6 +7425,60 @@ def equals(self, other):
                 return False
         return super().equals(other)
 
+    def explode(self, column, ignore_index=False):
```

Comment:
Ideally this implementation could be moved down to `frame` to share an implementation between `DataFrame` and `Series` as well.